Marlin/language_test.h

```diff
@@ -69,39 +69,39 @@
 
 // TESTSTRINGS
 
-#define STRG_ASCII_2 " !\"#$%&'()*+,-./"
-#define STRG_ASCII_3 "0123456789:;<=>?"
-#define STRG_ASCII_4 "@ABCDEFGHIJKLMNO"
-#define STRG_ASCII_5 "PQRSTUVWXYZ[\]^_"
-#define STRG_ASCII_6 "`abcdefghijklmno"
-#define STRG_ASCII_7 "pqrstuvwxyz{|}~"
-
-#define STRG_C2_8 ""
-#define STRG_C2_9 ""
-#define STRG_C2_a " ¡¢£¤¥¦§¨©ª«¬­®¯"
-#define STRG_C2_b "°±²³´µ¶·¸¹º»¼½¾¿"
-#define STRG_C3_8 "ÈÁÂÃÄÅÆÇÈÉÊËÌÍÎÏ"
-#define STRG_C3_9 "ÐÑÒÓÔÕÖ×ØÙÚÛÜÝÞß"
-#define STRG_C3_a "àáâãäåæçèéêëìíîï"
-#define STRG_C3_b "ðñòóôõö÷øùúûüýþÿ"
-
-#define STRG_D0_8 "ЀЁЂЃЄЅІЇЈЉЊЋЌЍЎЏ"
-#define STRG_D0_9 "АБВГДЕЖЗИЙКЛМНОП"
-#define STRG_D0_a "РСТУФХЦЧШЩЪЫЬЭЮЯ"
-#define STRG_D0_b "абвгдежзийклмноп"
-#define STRG_D1_8 "рстуфхцчшщъыьэюя"
-#define STRG_D1_9 "ѐёђѓєѕіїјљњћќѝўџ"
-#define STRG_D1_a "ѠѡѢѣѤѥѦѧѨѩѪѫѬѭѮѯ"
-#define STRG_D1_b "ѰѱѲѳѴѵѶѷѸѹѺѻѼѽѾѿ"
-
-#define STRG_E382_8 "よめもゃやゅゆょよらりるれろゎわ"
-#define STRG_E382_9 "ゐゑをんゔゕゖ゗゘゙゚゛ ゜ゝゞゟ"
-#define STRG_E382_a "゠ァアィイゥウェエォオカガキギク"
-#define STRG_E382_b "グケゲコゴサザシジスズセゼソゾタ"
-#define STRG_E383_8 "トチヂッツヅテデトドナニヌネノハ"
-#define STRG_E383_9 "バパヒビピフブプヘベペホボポマミ"
-#define STRG_E383_a "ムメモャヤュユョヨラリルレロヮワ"
-#define STRG_E383_b "ヰヱヲンヴヵヶヷヸヹヺ・ーヽヾヿ"
+#define STRG_ASCII_2 _UxGT(" !\"#$%&'()*+,-./")
+#define STRG_ASCII_3 _UxGT("0123456789:;<=>?")
+#define STRG_ASCII_4 _UxGT("@ABCDEFGHIJKLMNO")
+#define STRG_ASCII_5 _UxGT("PQRSTUVWXYZ[\]^_")
```

Comment:
You may need to use double-backslash to get a backslash (but I'm not sure with C strings).